bin/ci/deploy-dockerhub.sh

```diff
@@ -22,17 +22,19 @@ function retry() {
     return 0
 }
 
-# configure docker creds
-retry 3 docker login -e="$DOCKER_EMAIL" -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+if [[ "$DOCKER_DEPLOY" == true ]]; then
```

Comment:
The Antenna version doesn't have this. Is it good to keep these files in sync between projects?